libr/bp/bp.c

```diff
@@ -13,6 +13,12 @@ static void r_bp_item_free (RBreakpointItem *b) {
 	free (b->bbytes);
 	free (b->obytes);
 	free (b->module_name);
+        if (b->data) {
```

Comment:
use tabs, not spaces. and do not if(x)free(x); because free already checks for null